packages/dataviews/src/view-table.js

```diff
@@ -298,8 +298,7 @@ function TableRow( {
 				<td
 					className="dataviews-view-table__checkbox-column"
 					style={ {
-						width: 20,
-						minWidth: 20,
+						width: '1%',
```

Comment:
What does this change mean?

Reply:
Sorry I forgot to add a comment here. `width: 1%` applied to a table cell will make it 'hug' the contents regardless of width.